src/Components/Main.js

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import axios from 'axios';
-import { Container, Row, Col, Form, Button, Card, Image, Alert } from 'react-bootstrap';
+import CityForm from './Form';
+import CityAlert from './Alert';
+import CityCard from './Card';
+import { Container, Row, Col, Image } from 'react-bootstrap';
```

Comment:
I feel like this style of importing looks much cleaner in code! I am going to work this style into mine.